rotkehlchen/tests/api/blockchain/test_evm.py

```diff
@@ -73,7 +73,7 @@ def test_add_same_evm_account_for_multiple_chains(rotkehlchen_api_server):
 
     for chain in ('ETH', 'AVAX'):
```

Comment:
why not just use lower here, or even better serialize? Due to the `accounts_balances['assets'][chain]` part?

The insosistency persists here and is even trickier and confusing.

```diff
-accounts_balances['assets'][chain]
+accounts_balances['assets'][chain_native_token]
```

So the for should be iterating the chains and then getting the serialized name where needed and the serialized native token where needed.

Same thing should be done in many other tests around here. I will not be repeating this in the entire PR, but just do it so the confusion is gone.